ksql-rest-app/src/main/java/io/confluent/ksql/rest/client/KsqlRestClient.java

```diff
@@ -142,11 +144,16 @@ public void setServerAddress(String serverAddress) {
   }
 
   public RestResponse<KsqlEntityList> makeKsqlRequest(String ksql) {
+    RestResponse<KsqlEntityList> result;
     KsqlRequest jsonRequest = new KsqlRequest(ksql, localProperties);
     Response response = makePostRequest("ksql", jsonRequest);
-    KsqlEntityList result = response.readEntity(KsqlEntityList.class);
+    if (response.getStatus() == Response.Status.OK.getStatusCode()) {
```

Comment:
alternatively could do this in a `try{..}finally{..}` and do `return RestRespone....` etc. Removes the need for the variable.